core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -1435,6 +1435,14 @@ public Expression handle(Expression x) {
     }
   }
 
+  static Expression getDefaultValue(Type type, RoundingMode roundingMode) {
+    Primitive p = Primitive.of(type);
+    if (p != null) {
+      return Expressions.constant(p.defaultValue, type, roundingMode);
+    }
+    return Expressions.constant(null, type);
+  }
+
   static Expression getDefaultValue(Type type) {
```

Comment:
should the method without `roundingMode` be deprecated?

Reply:
Revert code.